lib/automations/release/branch-create-handler.js

```diff
@@ -328,6 +328,60 @@ const branchHandler = async ( context, octokit, config ) => {
 		issue_number: prCreated.data.number,
 		body: commentBody,
 	} );
+
+	// Get existing milestone
+	const milestone = await getMilestoneByTitle(
+		context,
+		octokit,
+		releaseVersion,
+		'open'
+	);
+	const milestoneNumber = milestone.number;
+	const milestoneTitle = milestone.title
+		.replace( '"', '' )
+		.replace( '[', '' )
+		.replace( ']', '' );
```

Comment:
Why do we need to do this?

You can also change to

```js
milestone.title.replace( /["\[\]]/g, '' );
```

if you prefer, the way you've done it might be more readable though.

Reply:
Thank you, I added the regex and updated the code. 👍